src/multibody/joint/joint-composite.hpp

```diff
@@ -73,6 +73,16 @@ namespace pinocchio
     typedef container::aligned_vector<JointDataVariant> JointDataVector;
     
     // JointDataComposite()  {} // can become necessary if we want a vector of JointDataComposite ?
+
+    JointDataCompositeTpl()
+    : joints()
+    , iMlast(0), pjMi(0)
+    , S(0)
+    , M(),v(),c()
+    , U(0,0), Dinv(0,0), UDinv(0,0)
```

Comment:
The dimensions are not all equal to 0 here.

Reply:
What would you prefer? I am using nv=0 as the default, since it is the same as the default JointModelComposite constructor.

Reply:
For me, UDinv and U are of dimension (6,0)

Reply:
Ah yes. ofcourse. I'll fix this.

Reply:
Done.